src/test/java/shwxr7/model/DiscConfigTests.java

```diff
@@ -13,11 +13,11 @@
 
 public class DiscConfigTests {
 
-  private final Element _10 = new Element("10", 10);
-  private final Element J = new Element("J", 11);
-  private final Element Q = new Element("Q", 12);
-  private final Element K = new Element("K", 13);
-  private final Element A = new Element("A", 0);
+  private final Element _10 = Element.of("1").get();
```

Comment:
If I am not mistaken, the name of _10 object, instead of "1"  should be "10".

Reply:
I misread the comment, sorry. I'll Fix this in an upcoming PR as well.

ty